src/taipy/core/_repository/_sql_repository.py

```diff
@@ -167,15 +176,15 @@ def _get_by_configs_and_owner_ids(self, configs_and_owner_ids):
         configs_and_owner_ids = set(configs_and_owner_ids)
 
         for config, owner in configs_and_owner_ids:
-            entry = self.__get_entities_by_config_and_owner(config.id, owner)
+            entry = self.__get_entity_by_config_and_owner(config.id, owner)
             if entry:
                 entity = self.__to_entity(entry)
                 key = config, owner
                 res[key] = entity
 
         return res
 
-    def __get_entities_by_config_and_owner(
+    def __get_entity_by_config_and_owner(
```

Comment:
```suggestion
    def __get_model_by_config_and_owner(
```
This method returns a model right?